financial-connections/src/main/java/com/stripe/android/financialconnections/presentation/FinancialConnectionsSheetNativeViewModel.kt

```diff
@@ -75,6 +85,26 @@ internal class FinancialConnectionsSheetNativeViewModel @Inject constructor(
     private val mutex = Mutex()
     val navigationFlow = navigationManager.navigationFlow
 
+    private val defaultTopAppBarState: TopAppBarState by lazy {
+        // The consent pane may choose to hide the Stripe logo. Therefore, let's hide it by default
+        // until the consent pane has received its payload with the show/hide information.
+        val forceHideStripeLogo = initialState.initialPane == Pane.CONSENT
```

Comment:
we can likely simplify this and default to hide the logo regardless of the first pane (just to avoid having pane specific logic on the activity level)

Reply:
Sounds good!

Reply:
I hard-coded `forceHideStripeLogo` to `true` for the first pane now, regardless of what pane it is. I’d argue that we still want it to avoid flashing the logo.